### PR TITLE
feat(j-s): court and defendants on every step in indictment cases

### PR DESCRIPTION
**Pull request by Arnarkari93**
[Asana ](https://app.asana.com/0/1199153462262248/1202868512043854/f)
## What

- Show court and defendant names on every step in indictment cases

## Screenshots / Gifs
Screenshot 2022-08-29 at 14 36 47

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/web/src/routes/Prosecutor/Indictments/Defendant.tsx [hunk 1]
```diff
@@ -250,7 +251,11 @@ const Defendant: React.FC = () => {
         <Box component="section" marginBottom={5}>
           <Box marginBottom={3}>
             <Text as="h3" variant="h3">
-              {capitalize(formatMessage(core.indictmentDefendant))}
+              {capitalize(
+                formatMessage(core.indictmentDefendant, {
+                  gender: Gender.MALE,
```
Comment (gudjong) [inline apps/judicial-system/web/src/routes/Prosecutor/Indictments/Defendant.tsx:256]: Should the gender be hardcoded?

File: apps/judicial-system/web/src/utils/hooks/useSections/index.ts [hunk 2]
```diff
@@ -243,7 +244,9 @@ const useSections = () => {
       children: [
         {
           type: 'SUB_SECTION',
-          name: capitalize(formatMessage(core.indictmentDefendant)),
+          name: capitalize(
+            formatMessage(core.indictmentDefendant, { gender: Gender.MALE }),
+          ),
```
Comment (gudjong) [inline apps/judicial-system/web/src/utils/hooks/useSections/index.ts:249]: Should the gender be hardcoded?

**Review by gudjong — CHANGES_REQUESTED** [review]

File: apps/judicial-system/web/src/routes/Prosecutor/Indictments/Defendant.tsx (hunk 1 above)
Comment (Arnarkari93) [reply apps/judicial-system/web/src/routes/Prosecutor/Indictments/Defendant.tsx:256]: I am not sure if we should be changing the title of the page and the sidebar when the genre changes. What do you think?

Comment (gudjong) [reply apps/judicial-system/web/src/routes/Prosecutor/Indictments/Defendant.tsx:256]: Not sure either. I will approve and we'll see if we get feedback.

**Review by gudjong — APPROVED** [review]

**Pull request merged**